src/app/loadout-drawer/subclass-drawer/AspectAndFragmentDrawer.tsx

```diff
@@ -0,0 +1,134 @@
+import { languageSelector } from 'app/dim-api/selectors';
+import { t } from 'app/i18next-t';
+import { PluggableInventoryItemDefinition } from 'app/inventory/item-types';
+import { isPluggableItem } from 'app/inventory/store/sockets';
+import PlugDrawer from 'app/loadout/plug-drawer/PlugDrawer';
+import _ from 'lodash';
+import React, { useCallback, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { SDDispatch } from './reducer';
+import { SelectedPlugs, SocketWithOptions } from './types';
+
+const MAX_ASPECTS = 2;
```

Comment:
We might just be able to ditch this and use the total number of aspect sockets. Currently that number is 2.